### PR TITLE
Test/hour

### PR DESCRIPTION
**Pull request by FilipePintoReis**
Tests for hour module

### CODE REVIEW

File: src/state.py [hunk 1]
```diff
@@ -18,6 +18,15 @@ def __str__(self):
             print(i, student)
         return st 
 
+    def set_score(self):
+        self.score = self.get_score()
```
Comment (LuisMMMTS) [inline src/state.py]: is this correct? insere um comentário aqui por favor para ficar mais claro o que está a acontecer

**Review by LuisMMMTS — DISMISSED** [review]

File: src/state.py (hunk 1 above)
Comment (FilipePintoReis) [reply src/state.py]: @LuisMMMTS This is meant to set the state score to a variable.
If you want to access the state score more than once, it's important that you do not call a costy function, like the heuristic, more than once.
For that purpose, the heuristic function is called once and stored for later use.

Comment (FilipePintoReis) [reply src/state.py]: Despite this, this was an error, as this function comes from a different PR which was accidentally merged here. I'm removing it for now !

**Review by LuisMMMTS — DISMISSED** [review]

**Comment by Jumaruba** [comment]
The add_schedule from state.py need to be changed to fit the changes recently done

**Review by Jumaruba — CHANGES_REQUESTED** [review]
The add_schedule from state.py need to be changed to fit the changes recently done.
It might be ok by doing:
```python
def add_schedule(self, subject, class_number, start_hour, end_hour, day):
    self.class_schedules[translate_subject_and_class(subject, class_number)] = Schedule(start_hour, end_hour, day)
```

**Review by Jumaruba — DISMISSED** [review]
The add_schedule from state.py need to be changed to fit the changes recently done

**Comment by FilipePintoReis** [comment]
@Jumaruba you still need to change the generator

**Review by Jumaruba — APPROVED** [review]

File: src/generator.py [hunk 2]
```diff
@@ -74,7 +74,7 @@ def time_generator(subjects, state):
     for number, subject in enumerate(subjects):
         for j in range(14): 
             hour_2 = Hour(number, j*2)
-            state.add_schedule(subject, j, hour_2, Hour(hour_2.hours, j*2+1))
+            state.add_schedule(subject, j, hour_2, Hour(hour_2.hours, j*2+1), "monday")
```
Comment (FilipePintoReis) [inline src/generator.py:77]: Isn't this hardcoded to be monday in every generated schedule?

Comment (FilipePintoReis) [reply src/generator.py:77]: @Jumaruba

**Comment by FilipePintoReis** [comment]
This has been merged in another pr already

**Pull request closed without merging**